approvals/src/main/java/no/sikt/nva/approvals/rest/ApprovalResponse.java

```diff
@@ -0,0 +1,42 @@
+package no.sikt.nva.approvals.rest;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeInfo.Id;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import java.net.URI;
+import java.util.Collection;
+import java.util.UUID;
+import no.sikt.nva.approvals.domain.Approval;
+import no.sikt.nva.approvals.domain.NamedIdentifier;
+import nva.commons.core.paths.UriWrapper;
+
+@JsonTypeInfo(use = Id.NAME, property = "type")
+@JsonTypeName("Approval")
+public record ApprovalResponse(
+    URI id,
+    UUID identifier,
+    Collection<NamedIdentifier> identifiers,
+    URI source,
+    String handle
+) {
+
+    private static final String APPROVAL_PATH = "approval";
+
+    public static ApprovalResponse fromApproval(Approval approval, URI requestUri) {
```

Comment:
When we make look up by handle or named identifier, do we want this uri to match it? Or do we always want
/approval/{uuid}

I think it should be static value like /approval/{uuid}, not depending on input variables